test/integration/fixtures/regression/1794/simple-ui.fixture.js

```diff
@@ -1,15 +1,22 @@
 'use strict';
 
-var path = '../../../../../lib/';
-var Mocha = require(path + 'mocha');
-var Test = require(path + 'test');
+var Mocha = require('../../../../../lib/mocha');
```

Comment:
[module-alias](https://www.npmjs.com/package/module-alias)?

Reply:
we can probably tackle that when we start transpiling stuff, but yes.